torch/csrc/inductor/aoti_torch/oss_proxy_executor.cpp

```diff
@@ -0,0 +1,266 @@
+#include <nlohmann/json.hpp>
+#include <fstream>
+#include <iostream>
+
+#include <torch/csrc/inductor/aoti_torch/oss_proxy_executor.h>
+
+using json = nlohmann::json;
+
+namespace {
+at::Tensor* tensor_handle_to_tensor_pointer(AtenTensorHandle handle) {
+  return reinterpret_cast<at::Tensor*>(handle);
+}
+} // namespace
+
+namespace torch {
+namespace aot_inductor {
+
+void OSSProxyExecutor::prefill_stack_with_static_arguments(
+    int index,
+    at::TypePtr schema_arg_type,
+    json serialized_arg,
```

Comment:
I've never used nlohmann before, but surely the `json` type isn't cheap to copy and therefore should be passed by const reference?